engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -673,6 +672,10 @@ Expression translateExpression(Tree tree, boolean isMethod) {
     }
   }
 
+  private static <T> T useOrElse(T one, T other) {
+      return one != null ? one : other;
+  }
```

Comment:
```suggestion
  private static <T> T useOrElse(T one, Supplier<T> other) {
      return one != null ? one : other.get();
  }
```

Reply:
This method is replacing the [one without supplier](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Objects.html#requireNonNullElse(T,T)) - there is also an alternative with supplier, but it is not used in `TreeToIr`.